src/modules/user/kyc/phone/sagas/sendCodeSaga.ts

```diff
@@ -17,7 +23,21 @@ export function* sendCodeSaga(action: SendCodeFetch) {
         yield put(sendCodeData());
         yield put(alertPush({message: ['success.phone.verification.send'], type: 'success'}));
     } catch (error) {
-        if (error.message.indexOf('resource.phone.number_exist') > -1) {
+        if (error.message.indexOf('resource.phone.exists') > -1) {
+            try {
+                yield call(API.post(sessionsConfig(getCsrfToken())), '/resource/phones/send_code', action.payload);
+                yield put(resendCodeData());
+                yield put(alertPush({ message: ['success.phone.verification.send'], type: 'success'}));
+            } catch (error) {
+                yield put(sendError({
+                    error,
+                    processingType: 'alert',
+                    extraOptions: {
+                        actionError: resendCodeError,
+                    },
+                }));
+            }
+        } else if (error.message.indexOf('resource.phone.number_exist') > -1) {
             yield put(alertPush({ message: error.message, type: 'success' }));
```

Comment:
```suggestion
            yield put(resendCode(action.payload))
```

Reply:
pushing alert 'resource.phone.number_exist' is useless, we have sendError action later in code that will do this job